src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/TakeCard.kt

```diff
@@ -0,0 +1,111 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIcon
+import de.jensd.fx.glyphs.materialdesignicons.MaterialDesignIconView
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.event.ActionEvent
+import javafx.geometry.Pos
+import javafx.scene.layout.AnchorPane
+import javafx.scene.layout.Pane
+import javafx.scene.layout.Priority
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.common.device.IAudioPlayer
+import org.wycliffeassociates.otter.jvm.app.UIColorsObject.Colors
+import tornadofx.*
+
+class TakeCard(take: Take, player: IAudioPlayer) : AnchorPane() {
+    private val badge = stackpane {
+        style {
+            backgroundColor += c(Colors["primary"])
```

Comment:
I believe @sarabiaj said that we don't want any of our widgets colors to be dependent on our project's resources?

Reply:
Should I just change them to default literal colors for now? Then we could apply our custom styles in the views

Reply:
Yes exactly!